test/support/http_method_compatibility.rb

```diff
@@ -11,7 +11,7 @@ class IntegrationTest < ActionDispatch::IntegrationTest
           if options.empty?
             super url
           else
-            super url, options
+            super url, **options
```

Comment:
only Rails 5+ are required to cope with Ruby2.7 and above (no need to change below part)